wesnoth-wof.po

```diff
@@ -5625,22 +5631,36 @@ msgid ""
 "The caravel can dock at coastal villages, though this makes it entirely "
 "vulnerable."
 msgstr ""
+"Pevné a spolehlivé karavely je základním prostředkem lidí, sloužícím "
+"k průzkumu oceánů, lodní přepravě a rozšiřování vlivu. Jsou lépe uzpůsobené "
+"plavbě po moři než lehčí plavidla, která zase bývají vídány při plavbě na "
+"hlubších řekách, kde manévrovací schopnosti a mělký ponor poněkud kompenzují "
+"jejich malou tonáž.\n"
```

Comment:
```suggestion
"Pevná a spolehlivá karavela je stěžejním prostředkem lidského "
"průzkumu oceánu, lodní přepravy a rozšiřování vlivu. Je lépe uzpůsobená "
"plavbě po moři než lehčí plavidla, přesto může být spatřena při plavbě na "
"hlubších řekách, kde manévrovací schopnosti a mělký ponor poněkud kompenzují "
"její tonáž.\n"
```

Reply:
Chápu to jinak, jen jako ospravedlnění toho, že může na řeku, přestože je karavela velká a určená pro moře. A může na řeku díky tomu, že se dá dobře ovládat a má malý ponor. Asi jde o to říct, přes jaké hexy se může posouvat (mělká voda? - nevím, nehrál jsem).